server/src/main/java/org/elasticsearch/indices/recovery/RecoveryState.java

```diff
@@ -343,7 +343,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         static final String REUSED = "reused";
         static final String REUSED_IN_BYTES = "reused_in_bytes";
         static final String PERCENT = "percent";
-        static final String DETAILS = "details";
+        static final String DETAILED = "detailed";
```

Comment:
let's leave this field as is. "details" in the output is nicer, and we don't break existing parsers.